### PR TITLE
[new release] merlin (3.2.2)

### PR DESCRIPTION
**Pull request by trefis**
Installation with Opam

- Project page: <a href="https://github.com/ocaml/merlin">https://github.com/ocaml/merlin</a>

##### CHANGES:

Tue Oct  9 11:25:12 BST 2018

Update cmt magic number for 4.07.1

### CODE REVIEW

**Comment by trefis** [comment]
x-ref: https://github.com/ocaml/merlin/issues/869

File: packages/merlin/merlin.3.2.2/opam [hunk 1]
```diff
@@ -0,0 +1,76 @@
+opam-version: "2.0"
+name: "merlin"
+version: "3.1.0"
```
Comment (kit-ty-kate) [inline packages/merlin/merlin.3.2.2/opam]: This is wrong. This line should be removed

Comment (trefis) [reply packages/merlin/merlin.3.2.2/opam]: Indeed. That'll teach me, what was I think relying on tools for that sort of things.

File: packages/merlin/merlin.3.2.2/opam [hunk 2]
```diff
@@ -1,6 +1,6 @@
 opam-version: "2.0"
 name: "merlin"
-version: "3.1.0"
+version: "3.2.2"
```
Comment (mseri) [inline packages/merlin/merlin.3.2.2/opam]: This is no longer used. Please remove it from the opam file

Comment (mseri) [reply packages/merlin/merlin.3.2.2/opam]: Don't worry about it, I can do it easily from here

**Comment by mseri** [comment]
We should probably "sqash and merge" this to avoid cluttering the history with fix commits

**Comment by trefis** [comment]
Sounds good to me.

And thanks, I'll remove the version field from the opam file we have in our repo.

**Comment by kit-ty-kate** [comment]
The tests fail :/
```
#=== ERROR while compiling merlin.3.2.2 =======================================#
# context     2.0.0 | linux/x86_64 | ocaml-base-compiler.4.07.0 | file:///home/travis/build/ocaml/opam-repository
# path        ~/.opam/4.07.0/.opam-switch/build/merlin.3.2.2
# command     ~/.opam/opam-init/hooks/sandbox.sh build dune runtest -p merlin -j 1
# exit-code   1
# env-file    ~/.opam/log/merlin-15322-0e81fa.env
# output-file ~/.opam/log/merlin-15322-0e81fa.out
### output ###
# [...]
# -    "notifications": []
# -  }
# +  ../merlin-wrapper: 3: ../merlin-wrapper: Bad substitution
# +  ../merlin-wrapper: 5: ../merlin-wrapper: realpath: not found
# +  ../merlin-wrapper: 7: ../merlin-wrapper: Bad substitution
# +  ../merlin-wrapper: 7: ../merlin-wrapper: /../../install/default/bin/ocamlmerlin: not found
#  
#    $ $MERLIN server stop-server
# +  ../merlin-wrapper: 3: ../merlin-wrapper: Bad substitution
# +  ../merlin-wrapper: 5: ../merlin-wrapper: realpath: not found
# +  ../merlin-wrapper: 7: ../merlin-wrapper: Bad substitution
# +  ../merlin-wrapper: 7: ../merlin-wrapper: /../../install/default/bin/ocamlmerlin: not found
```

**Comment by trefis** [comment]
Yes.
I'm not about to fix that :)

**Comment by trefis** [comment]
Ping.

**Comment by kit-ty-kate** [comment]
Thanks!

**Pull request merged**